docs/DeveloperGuide.adoc

```diff
@@ -998,7 +994,6 @@ due to its key size.
 
 We believe that both alternatives have reasonable pros and cons and choosing either option would be beneficial to `**THRIFT**`.
 The added data security will assure the user that their data is safe and allows them to use `**THRIFT**` with a peace of mind.
-
 // end::dataencryption[]
 
```

Comment:
Under **Aspect: Uniqueness of Tag objects within a Transaction** alternative 1's cons:

... as they are **no** longer...

Reply:
Fixed.